### PR TITLE
Mentor follow blindness fix

### PR DESCRIPTION
**Pull request by Bilbo367**
## About The Pull Request

Mentor follow is no longer affected by screen affects like blindness, and color effects like color blindness.

## Changelog

:cl:
qol: mentor follows no longer are affected by blindness
/:cl:

### CODE REVIEW

File: fulp_modules/features/mentors/follow.dm [hunk 1]
```diff
@@ -22,3 +27,8 @@
 	to_chat(GLOB.admins, span_adminooc("<span class='prefix'>MENTOR:</span> <EM>[key_name(usr)]</EM> is no longer following <EM>[key_name(mentor_datum.following)]</span>"))
 	log_mentor("[key_name(usr)] stopped following [key_name(mentor_datum.following)]")
 	mentor_datum.following = null
+	//Add back mentor's blindness/color blindness
+	usr.reload_fullscreen()
```
Comment (JohnFulpWillard) [inline fulp_modules/features/mentors/follow.dm:31]: If you are cured of blindness between this do you keep it lol

Comment (JohnFulpWillard) [reply fulp_modules/features/mentors/follow.dm:31]: I’ve never looked into this stuff much but it looks dangerous

Comment (Bilbo367) [reply fulp_modules/features/mentors/follow.dm:31]: > If you are cured of blindness between this do you keep it lol

Yes

Comment (JohnFulpWillard) [reply fulp_modules/features/mentors/follow.dm:31]: Doesn't it make this not good? If I get my eyes stabbed out while mfollowing someone then I am permablind and would need admins to know what caused it and know how to fix it

Comment (Bilbo367) [reply fulp_modules/features/mentors/follow.dm:31]: I miss understood. No you would be fine. This is not true for color blindness trying to fix it now.

**Comment by Bilbo367** [comment]
The way I see it we have 2 paths in front of us. A: Merge as is if you lose what's giving you color effects your permanently with that color. B: I remove the part that touches the colors. Still better then base since we be able to see.

**Pull request closed without merging**